Services/CheckRightsService/Commands/RemoveRightsFromUserCommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.CheckRightsService.Commands.Interfaces;
+using LT.DigitalOffice.CheckRightsService.Models;
+using LT.DigitalOffice.CheckRightsService.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LT.DigitalOffice.CheckRightsService.Commands
+{
+    public class RemoveRightsFromUserCommand : IRemoveRightsFromUserCommand
+    {
+        private readonly ICheckRightsRepository repository;
+        private readonly IValidator<RemoveRightsFromUserRequest> validator;
+
+        public RemoveRightsFromUserCommand(
+            [FromServices] ICheckRightsRepository repository,
+            [FromServices] IValidator<RemoveRightsFromUserRequest> validator)
+        {
+            this.repository = repository;
+            this.validator = validator;
+        }
+
+        public void Execute(RemoveRightsFromUserRequest request)
+        {
+            validator.ValidateAndThrow(request);
+            repository.RemoveRightsFromUser(request);
+
+            // TODO: Check rights of user (sender) with Kernel_AccessValidator
+        }
```

Comment:
Will you add check rights in the this pr?

Reply:
Fixed.